onnxruntime/core/optimizer/constant_folding.cc

```diff
@@ -7,63 +7,82 @@
 #include "core/framework/op_kernel.h"
 #include "core/framework/ml_value.h"
 
+using namespace ::onnxruntime::common;
```

Comment:
>:: [](start = 16, length = 2)

The '::' prefix should never be necessary in our code (we should never have an onnxruntime namespace nested within the top level onnxruntime namespace).